molecule/templates/test.j2

```diff
@@ -0,0 +1,2 @@
+this is a {{ test }}
+
```

Comment:
I could pass in another directory under molecule/molecule though. So instead of having it in molecule/templates it could be in like, molecule/tests or testdata or whatever.